src/data-mgt/node/utils/date.js

```diff
@@ -74,13 +74,12 @@ const addMonthsToProvidedDate = (date, number) => {
 
 const isDate = (date) => {
   try {
-    logText("checking the date");
-    return date instanceof Date && !isNaN(date);
+    return date.includes("-") || date.includes("/");
```

Comment:
new approach for validating dates. Main part of the PR. Previous method was buggy.